website/content/docs/agent/wal-logstore/enable.mdx

```diff
@@ -39,113 +16,3 @@ The following issues exist in Consul 1.15.0 and 1.15.1.
  * A follower that is disconnected may be unable to catch up if it is using the WAL backend.
  * Restoring user snapshots can break replication to WAL-enabled followers.
  * Restoring user snapshots can cause a WAL-enabled leader to panic.
-
-## Risks
-
-While their likelihood remains low to very low, be aware of the following risks before implementing the WAL backend:
-
- - If WAL corrupts data on a Consul server agent, the server data cannot be recovered. Restart the server with an empty data directory and reload its state from the leader to resolve the issue.
- - WAL may corrupt data or contain a defect that causes the server to panic and crash. WAL may not restart if the defect recurs when WAL reads from the logs on startup. Restart the server with an empty data directory and reload its state from the leader to resolve the issue.
- -  If WAL corrupts data, clients may read corrupted data from the Consul server, such as invalid IP addresses or unmatched tokens. This outcome is unlikely even if a recurring defect causes WAL to corrupt data because replication uses objects cached in memory instead of reads from disk. Restart the server with an empty data directory and reload its state from the leader to resolve the issue.
- - If you enable a Consul CE server to use WAL or enable WAL on a voting server with Consul Enterprise, WAL may corrupt the server's state, become the leader, and replicate the corrupted state to all other servers. In this case, restoring from backup is required to recover a completely uncorrupted state. Test WAL on a non-voting server in Enterprise to prevent this outcome. You can add a new non-voting server to the cluster to test with if there are no existing ones.
-
-## Enable log verification
-
-You must enable log verification on all voting servers in Enterprise and all servers in CE because the leader writes verification checkpoints.
-
-1. On each voting server, add the following to the server's configuration file:
-
-  ```hcl
-  raft_logstore {
-    verification {
-      enabled = true
-      interval = "60s"
-    }
-  }
-  ```
-
-1. Restart the server to apply the changes. The `consul reload` command is not sufficient to apply `raft_logstore` configuration changes.
-1. Run the `consul operator raft list-peers` command to wait for each server to become a healthy voter before moving on to the next. This may take a few minutes for large snapshots.
-
-When complete, the server's logs should contain verifier reports that appear like the following example:
-
-```log hideClipboard
-2023-01-31T14:44:31.174Z [INFO]  agent.server.raft.logstore.verifier: verification checksum OK: elapsed=488.463268ms leaderChecksum=f15db83976f2328c rangeEnd=357802 rangeStart=298132 readChecksum=f15db83976f2328c
-```
```

Comment:
Isn't this section on log verification still relevant info even when WAL is defaulted on?

Reply:
Log verification is enabled by default now as part of WAL. The reasoning is that it have minimal impact but great benefits in case of bugs. 

I will double check if it's documented as part of the logstore config properly.